examples/generative/corrdiff/train.py

```diff
@@ -246,7 +253,7 @@ def main(cfg: DictConfig) -> None:
     logger0.info(f"Preconditioning & loss:  {precond}")
     logger0.info(f"Number of GPUs:          {dist.world_size}")
     logger0.info(f"Batch size:              {c.batch_size_global}")
-    logger0.info(f"Mixed-precision:         {c.network_kwargs.use_fp16}")
```

Comment:
Should we also remove it from here? https://github.com/NVIDIA/modulus/blob/b10fe30c8bb18aade11650817731ece8fe95ec48/examples/generative/corrdiff/train.py#L218

Reply:
(Later we can discuss whether it makes sense to keep the existing FP16 mode which does quite a bit of manual configuration inside the U-Nets. Removing it could simplify the U-Net code nicely. But I wouldn't try to remove it in this PR as this requires feedback from other devs).